CMSgov/mint-app
# [Easi-1999] Model Plan Status Changes

# EASI-1999
## Changes and Description
- Created UI for displaying model plan status
- Created a route and UI for editing status
- Created GQL mutation and types for modifying status
- Testing where applicable



## How to test this change

1. Create a new plan and navigate to the task list page
2. Confirm that the status is displayed on top of page ![image](https://user-images.githubusercontent.com/2349702/168639161-8d5485d4-cf4e-4feb-8af3-ee9c81d50cef.png)
3. Click `Update` link will navigate user to status updating page. Ensure this page matches figma
4. Clicking on dropdown has the same options as figma 
5. Click `Update` will return user to task list page and update status
6. Ensure click `Don’t update status and return to task list` doesn't update and returns to list



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
